src/sentry_sync.h

```diff
@@ -8,8 +8,57 @@
 
 // define a recursive mutex for all platforms
 #ifdef SENTRY_PLATFORM_WINDOWS
-#    include <synchapi.h>
+#    if _WIN32_WINNT >= 0x0600
+#        include <synchapi.h>
+#    else
+#        include <windows.h>
```

Comment:
do we need this? I think `sentry_boot` already pulls in a minimal version of `windows.h`.

Reply:
deleted